src/main/java/kr/co/vcnc/haeinsa/exception/ConflictException.java

```diff
@@ -37,4 +37,8 @@ public ConflictException(String message) {
     public ConflictException(String message, Throwable cause) {
         super(message, cause);
     }
+
+    public ConflictException(String message, ConflictExceptionLog log) {
+        super(message + " [ExceptionLog]:" + log);
```

Comment:
There is transaction's data(TRowLock has mutations.) in ConflictExceptionLog.
In my opinion, making ConflictExceptionLog into a exceptions' variable is more secure than appending it to exception message.